package.json

```diff
@@ -32,12 +32,12 @@
         "serve:mock-axe-android": "http-server -p 9051 -e json ./src/tests/miscellaneous/mock-axe-android",
         "start:electron": "electron drop/electron/electron/product/bundle/main.bundle.js",
         "start:electron:dev": "cross-env DEV_MODE=true electron drop/electron/electron/product/bundle/main.bundle.js",
-        "test": "jest --projects src/tests/unit",
-        "test:e2e": "jest --projects src/tests/end-to-end --runInBand",
+        "test": "cross-env --max-old-space-size=16384 jest --projects src/tests/unit",
```

Comment:
is this necessary for updating node version?

Reply:
Can be a separate PR, but this looks like too little of a change to create a new PR; that’s why included this here.